src/tests/ftest/pool/RebuildTests.py

```diff
@@ -52,14 +51,39 @@ class RebuildTests(Test):
     def setUp(self):
         self.hostlist = None
         # get paths from the build_vars generated by build
-        with open('../../../.build_vars.json') as f:
-            self.build_paths = json.load(f)
+        with open('../../../.build_vars.json') as file_ds:
+            self.build_paths = json.load(file_ds)
 
         # setup the DAOS python API
-        self.Context = DaosContext(self.build_paths['PREFIX'] + '/lib/')
+        self.context = DaosContext(self.build_paths['PREFIX'] + '/lib/')
+        self.createuid = os.geteuid()
+        self.creategid = os.getegid()
+        # parameters used in pool create that are in yaml
+        self.createmode = self.params.get("mode", '/run/testparams/createmode/')
+        self.createsetid = self.params.get("setname", '/run/testparams/createset/')
+        self.createsize = self.params.get("size", '/run/testparams/createsize/')
+        # how many objects and records are we creating
+        self.objcount = self.params.get("objcount", '/run/testparams/numobjects/*')
+        self.reccount = self.params.get("reccount", '/run/testparams/numrecords/*')
```

Comment:
(pylint-line-too-long) Line too long (83/80)

Reply:
no belong to current patch.